Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -2493,7 +2493,7 @@ script:
     - contextPath: ServiceNow.Tasks.State
       description: state of task
       type: Unknown
-  dockerimage: demisto/python3:3.9.9.25564
+  dockerimage: demisto/python3:3.10.1.25933
```

Comment:
Let's revert the docker change for the time being (until the issue will be fixed)